controller/src/main/java/com/epam/esm/controller/controllers/TagController.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.esm.controller.controllers;
+
+import com.epam.esm.controller.util.HateoasBuilder;
+import com.epam.esm.controller.util.PaginationBuilder;
+import com.epam.esm.entity.TagDto;
+import com.epam.esm.service.TagService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.hateoas.PagedModel;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v2/tags")
+public class TagController {
+
+    public static final String DEFAULTE_PAGE_VALUE = "1";
+    public static final String DEFAULTE_SIZE_VALUE = "10";
+
+    @Autowired
+    private final TagService tagService;
+    private final HateoasBuilder hateoas;
+    private final PaginationBuilder<TagDto> pagination;
+
+    public TagController(TagService tagService, HateoasBuilder hateoas, PaginationBuilder<TagDto> pagination) {
```

Comment:
Suggest to change "hateos" -> "hateoasBuilder",  "pagination" -> "paginationBuilder".

Reply:
hateos and pagination were renamed everywhere